files/zh-cn/web/api/navigator/mediacapabilities/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Navigator：mediaCapabilities 属性
+slug: Web/API/Navigator/mediaCapabilities
+l10n:
+  sourceCommit: acfe8c9f1f4145f77653a2bc64a9744b001358dc
+---
+
+{{APIRef("HTML DOM")}}
+
+{{domxref("Navigator")}} 接口的 **`Navigator.mediaCapabilities`** 只读属性返回一个 {{domxref("MediaCapabilities")}} 对象，该对象可以公开有关特定格式的解码和编码能力以及由 [媒体功能 API](/zh-CN/docs/Web/API/Media_Capabilities_API) 定义的输出能力的信息。
```

Comment:
https://www.w3.org/zh-hans/news/2020/first-public-working-drafts-media-capabilities-picture-in-picture-media-session-standard/

```suggestion
{{domxref("Navigator")}} 接口的 **`Navigator.mediaCapabilities`** 只读属性返回一个 {{domxref("MediaCapabilities")}} 对象，该对象可以暴露有关给定格式的解码和编码能力以及由[媒体能力 API](/zh-CN/docs/Web/API/Media_Capabilities_API) 定义的输出能力的信息。
```